libbeat/common/kubernetes/types.go

```diff
@@ -116,3 +116,17 @@ func ContainerIDWithRuntime(s PodContainerStatus) (string, string) {
 	}
 	return "", ""
 }
+
+func EventLastTimestamp(eve *Event) metav1.Time {
+	lastTimestamp := eve.LastTimestamp
+	if lastTimestamp.IsZero() {
+		if eve.Series != nil && !eve.Series.LastObservedTime.IsZero() {
+			lastTimestamp = metav1.Time(eve.Series.LastObservedTime)
+		} else if !eve.EventTime.IsZero() {
+			lastTimestamp = metav1.Time(eve.EventTime)
```

Comment:
Here we fallback to a value that is described as `Time when this Event was first observed`, is this something that is actually correct? `EventTime` is mostly a good fallback if `FirstTimestamp` is not present.